### PR TITLE
patch for submit closed question

### LINKED ISSUE #1: Submitting a closed question will detele uploaded file
Submitting a closed question will detele uploaded file but can't reupload, Maybe some protection should be done?

### PR DESCRIPTION
**Pull request by kryr34**

### CODE REVIEW

File: hwp.py [hunk 1]
```diff
@@ -86,6 +86,14 @@ def show_status(q_status: dict, t_status: dict) -> None:
...
                 index = args[2].rjust(3, '0')
                 file_path = args[3]
+                q_status = s.get_question_statuses()[index]
+                if q_status['release_status'] == "Closed" :
+                    print("The question has closed, "\
+                          "submitting the file anyways will delete the uploaded file "\
+                          "but won't upload your local file."\
+                          "Do you wish to continue? [y/N]", end='')
+                    if input() != "y":
+                        exit(0)
```
Comment (NOOBDY) [inline hwp.py:96]: probably should add `print("Aborting")`

Comment (NOOBDY) [inline hwp.py:96]: probably should add `print("Aborting")` or some message

**Review by NOOBDY — CHANGES_REQUESTED** [review]

**Comment by NOOBDY** [comment]
@Dragonfly911117 this also needs to be implemented in the GUI version cause it's not handled in the backend

File: hwp.py (hunk 1 above)
Comment (kryr34) [reply hwp.py:96]: no

**Pull request merged**